centreon/www/include/configuration/configObject/hostgroup_dependency/listHostGroupDependency.php

```diff
@@ -43,12 +46,7 @@
 
 $list = $_GET["list"] ?? null;
 
-$aclCond = "";
-if (!$oreon->user->admin) {
-    $aclCond = " AND hostgroup_hg_id IN ($hgstring) ";
-}
-
-$search = \HtmlAnalyzer::sanitizeAndRemoveTags(
+$search = HtmlAnalyzer::sanitizeAndRemoveTags(
```

Comment:
deprecated you can use HtmlSanitizer 
```suggestion
$search = HtmlSanitizer::createFromString($_POST['searchHGD'] ?? $_GET['searchHGD'] ?? null)->removeTags()->sanitize()->getString();
```